farm/evaluation/metrics.py

```diff
@@ -87,18 +102,19 @@ def compute_metrics(metric, preds, labels):
 
 
 def compute_report_metrics(head, preds, labels):
-    if head.ph_output_type == "per_token":
+    if head.ph_output_type in registered_reports:
+        report_fn = register_report[head.ph_output_type]
```

Comment:
I believe there is a typo here, shouldn't this be `registered_reports[head.ph_output_type]` instead of `register_report`? I think the bug was present in my version, too!

Reply:
Aside from that, looks good to me and you can definitely merge when it's fixed! Thanks again!

Reply:
Oh yeah, good catch. I will fix it & merge.